src/components/App.js

```diff
@@ -1,27 +1,39 @@
 import React from 'react';
 import LearningModule from './LearningModule';
+import HomePage from './HomePage';
+import Catalog from './Catalog';
 import SignIn from './auth/SignIn';
 import SignUp from './auth/SignUp';
 import NavBar from './common/NavBar';
-import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import '../App.css';
 
 function App() {
   return (
     <Router>
       <div className="App">
-        <NavBar brandTitle="Node Warrior" />
-          <Switch>
-            <Route path="/signin">
-              <SignIn />
-            </Route>
-            <Route path="/signup">
-              <SignUp />
-            </Route>
-            <Route path="/">
-              {/* TODO going to put some form of routing here eventually */}
-              <LearningModule />
-            </Route>
+        <Switch>
+          <Route path="/signin">
+            <SignIn />
+          </Route>
+          <Route path="/signup">
+            <SignUp />
+          </Route>
+          <Route path="/module">
+            <LearningModule
+              navBar={<NavBar navBarType="module" />}
+            />
+          </Route>
+          <Route path="/catalog">
+            <Catalog
+              navBar={<NavBar navBarType="catalog" />}
+            />
+          </Route>
+          <Route path="/">
+            <HomePage
+              navBar={<NavBar navBarType="homepage" />}
```

Comment:
Just a thought, you could make Homepage take a "navBarType" prop. Usually you want to have props be as simple as possible, so the string is probably better than the whole node. Either is fine tho, you choose

Reply:
If it would end up adding a bunch of props down the line, then just pass the node, but sicne NavBar only takes one prop, passing that might be simpler.